apps/core/tailwind.config.js

```diff
@@ -129,6 +130,19 @@ module.exports = {
       height: {
         header: "68px",
       },
+      transitionTimingFunction: {
+        "ease-in-out-cubic": "cubic-bezier(0.65, 0, 0.35, 1)",
+        "ease-out-cubic": "cubic-bezier(0.33, 1, 0.68, 1)",
+      },
+      transitionDuration: {
+        400: "400ms",
+      },
+      backgroundImage: {
+        noMedia: "linear-gradient(165.96deg, #e6f5ff 9.97%, #ebecff 94.97%)",
```

Comment:
In the figma this is called placeholderGradient01, can we match that name? Also maybe just round these to 10% and 95% ?